internal/cephfs/errors.go

```diff
@@ -62,3 +62,33 @@ func (e ErrVolumeNotFound) Error() string {
 func (e ErrVolumeNotFound) Unwrap() error {
 	return e.err
 }
+
+// ErrCloneInProgress for cloned subvolume.
+type ErrCloneInProgress struct {
```

Comment:
No need to add these full types, just use sentinel errors to make things simpler

Reply:
Thanks for creating standard error values for many, only `ErrCloneInProgress` is left to do (here). The error message is never logged without additional details, so there is no need for `ErrCloneInProgress` to be special. Please make it a sentinel error like the others and keep things simple.

Reply:
Its already taken care. Thanks.

Reply:
Could you please squash/fixup those changes that undoes this error type? There is no need to leave traces in the git history for the unintentional adding and then reverting such things.